src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -293,3 +297,20 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
 
     with self.command_group('acr token credential', acr_token_credential_generate_util) as g:
         g.command('generate', 'acr_token_credential_generate')
+
+    with self.command_group('acr private-endpoint-connection', acr_private_endpoint_connection_util,
```

Comment:
we're using command to reflect custom command.  I'm fine 😄 @Juliehzl

Reply:
Hey @MyronFanQiu, I am not sure what you meant here. Please can you review / comment on #12381